src/compiler/parser.ts

```diff
@@ -4615,6 +4729,12 @@ module ts {
             return isIdentifierOrKeyword() || token === SyntaxKind.StringLiteral;
         }
 
+        function nextTokenIsIdentifierOrKeywordOrStringLiteralOrAsteriskOrOpenBrace() {

```

Comment:
i think i'd prefer this just be called "nextTokenIsImportDeclarationStart" or something like that.